Sources/SwiftFormat/PrettyPrint/TokenStreamCreator.swift

```diff
@@ -3991,14 +4054,14 @@ class CommentMovingRewriter: SyntaxRewriter {
   }
 
   override func visit(_ node: CodeBlockItemSyntax) -> CodeBlockItemSyntax {
-    if shouldFormatterIgnore(node: Syntax(node)) {
+    if shouldFormatterIgnore(node: Syntax(node)) || !Syntax(node).isInsideSelection(selection) {
```

Comment:
Don’t we need to perform the `isInsideSelection` check for all syntax nodes instead of just `CodeBlockItemSyntax` and `MemberBlockItemSyntax`?

Reply:
Hmm... maybe. Lemme see if I can come up with a scenario where it'll matter.

Reply:
While this PR is pretty close to being ready, this is still an outstanding question....

Reply:
This aspect of the PR isn't working correctly yet. We're basically disabling all rewriting rules when there's a selection. 🙁 Looking into this now.